src/ast/irbuilderbpf.cpp

```diff
@@ -2511,11 +2513,7 @@ void IRBuilderBPF::CreateHelperErrorCond(Value *return_value,
   // return int and we need to use Int32Ty value to check if a return
   // value is negative. TODO: use Int32Ty as a return type
```

Comment:
Is this TODO still relevant?

Reply:
Yeah, we're still setting the ret for most helpers to Int64Ty